docs/proposals/parameter-distribution.md

```diff
@@ -0,0 +1,153 @@
+
+### Goals
+- Add `Distribution` field to `ParameterSpec` alongside `ParameterType`.
+- Support for the log-uniform, normal, and log-normal Distributions.
+- Update the Experiment and gRPC API to support `Distribution`.
+- Update logic to handle the new parameter distributions for each suggestion service (e.g., Optuna, Hyperopt).
+- Extend the Python SDK to support the new `Distribution` field.
+### Non-Goals
+- This proposal do not aim to create new version for CRD APIs.
+- No changes will be made to the core optimization algorithms beyond supporting new distributions.
+
+## Proposal
+
+### Parameter Distribution Comparison Table
+| Distribution Type             | Hyperopt              | Optuna                               | Ray Tune              |
+|-------------------------------|-----------------------|--------------------------------------|-----------------------|
+| **Uniform Continuous**        | `hp.uniform`          | `FloatDistribution`                  | `tune.uniform`        |
+| **Quantized Uniform**         | `hp.quniform`         | `DiscreteUniformDistribution`(deprecated)  Use `FloatDistribution` instead. | `tune.quniform`       |
+| **Log Uniform**               | `hp.loguniform`       | `LogUniformDistribution`(deprecated)  Use `FloatDistribution` instead. | `tune.loguniform`     |
+| **Uniform Integer**           | `hp.randint` or quantized distributions with step size `q` set to 1 | `IntDistribution`                    | `tune.randint`        |
+| **Categorical**               | `hp.choice`           | `CategoricalDistribution`            | `tune.choice`         |
+| **Quantized Log Uniform**     | `hp.qloguniform`      | Custom Implementation Use `FloatDistribution` instead.                 | `tune.qloguniform`    |
+| **Normal**                    | `hp.normal`           | (Not directly supported)             | `tune.randn`          |
+| **Quantized Normal**          | `hp.qnormal`          | (Not directly supported)             | `tune.qrandn`         |
+| **Log Normal**                | `hp.lognormal`        | (Not directly supported)             | (Use custom transformation in `tune.randn`) |
+| **Quantized Log Normal**      | `hp.qlognormal`       | (Not directly supported)             | (Use custom transformation in `tune.qrandn`) |
+| **Quantized Integer**         | `hp.quniformint`      | `IntUniformDistribution`(deprecated)  Use `IntDistribution` instead.          |                       |
+| **Log Integer**               |                       | `IntLogUniformDistribution`(deprecated)  Use `IntDistribution` instead.          | `tune.lograndint`     |
+
+### How is Nevergrad implementing Hyperopt?
+Nevergrad maps parameter types (like p.Scalar, p.Log, p.Choice, etc.) from Nevergrad to corresponding Hyperopt search space definitions (hp.uniform, hp.loguniform, hp.choice, etc.).
```

Comment:
In the last sync, we discussed Nevergrad, so I looked into it, Nevergrad maps parameter types to corresponding Hyperopt search space definitions and then constructs the complete search space. 
ref: https://github.com/facebookresearch/nevergrad/blob/c23f20406d9223afbe253f9807ca7e8ba993a1ac/nevergrad/optimization/externalbo.py#L47

Reply:
Instead of this dedicated section for the Nevergrad, could you just add the Nevergrad column to the corresponding table above?

Reply:
Removed the Nevergrad section and added a column for it in the comparison table.